site/_docs/history.md

```diff
@@ -29,18 +29,94 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 <!--
-{% comment %}
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.33.0">1.33.0</a> / YYYY-MM-DD
-{: #v1-33-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / XXXX-XX-XX
+{: #v1-34-0}
 
-This release comes x months after [1.32.0](#v1-32-0),
+This release comes x months after [1.33.0](#v1-33-0),
 contains contributions from x contributors, and resolves x issues.
 
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-34-0}
+
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 18;
+Guava versions 16.0.1 to 31.1-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-34-0}
+
+#### Dependency version upgrade
+{: #dependency-1-34-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-34-0}
+
+#### Build and test suite
+{: #build-1-34-0}
+
+#### Web site and documentation
+{: #site-1-34-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.33.0">1.33.0</a> / 2023-02-04
+{: #v1-33-0}
+
+This release comes five months after [1.32.0](#v1-32-0),
```

Comment:
As @zstan pointed out in [CALCITE-5253](https://issues.apache.org/jira/browse/CALCITE-5253), the Jira tickets closed with status `Closed` instead of `Resolved` seems to have been left out (I looked all those appearing [here](https://issues.apache.org/jira/browse/CALCITE-5372?jql=project%20%3D%20Calcite%20AND%20fixVersion%20%3D%201.33.0%20AND%20status%20%3D%20Closed) and none of them is included in the release note).

Is there a script to compile the release notes or did you run a JQL command of yours, @jbalint?

We should probably always use `Resolved` status for consistency instead of `Closed`, but if a script exists somewhere, we might make it a bit more flexible in this respect.

Reply:
We are using the git log for compiling the release notes; not JIRA.

There is a general convention to mark the JIRA status as `Resolved` when merging a fix. It is transitioned to `Closed` after the release by the RM (https://calcite.apache.org/docs/howto.html#making-a-release-candidate).

Reply:
Thanks @zabetak for chiming in, all clear now.

Then I think that CALCITE-5253 has been simply missed in the release notes.

Reply:
added to release notes